taxi/models.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        verbose_name = "manufacturer"
+        verbose_name_plural = "manufacturers"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+    username = models.CharField(max_length=255, unique=True)
+    email = models.CharField(max_length=255)
+    password = models.CharField(max_length=255)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
```

Comment:
Redefining fields like `username`, `email`, `password`, `first_name`, and `last_name` in the `Driver` model is unnecessary since these fields are already defined in `AbstractUser`. Consider removing these lines to avoid redundancy.